### PR TITLE
Angle: Improved code quality

### PR DESCRIPTION
**Pull request by saeeedtorabi**
- fixed potential bugs
- improved src code formatting
- improved documentation
- improved tests

### CODE REVIEW

**Review by pmconne — APPROVED** [review]
Approved /docs/ changes.

File: core/geometry/src/geometry3d/AngleSweep.ts [hunk 1]
```diff
@@ -101,32 +134,39 @@ export class AngleSweep implements BeJSONFunctions {
...
+    this._radians1 = other._radians1;
+  }
   /** create a full circle sweep (CCW). startRadians defaults to 0 */
   public static create360(startRadians?: number): AngleSweep {
     startRadians = startRadians ? startRadians : 0.0;
     return new AngleSweep(startRadians, startRadians + 2.0 * Math.PI);
   }
-  /** create a sweep from the south pole to the north pole. */
-  public static createFullLatitude() { return AngleSweep.createStartEndRadians(-0.5 * Math.PI, 0.5 * Math.PI); }
+  /** create a sweep from the south pole to the north pole (-180 to +180). */
```
Comment (dassaf4) [inline core/geometry/src/geometry3d/AngleSweep.ts:158]: Comment should be "(-90 to +90)".

Comment (saeeedtorabi) [reply core/geometry/src/geometry3d/AngleSweep.ts:158]: will fix

File: core/geometry/src/geometry3d/AngleSweep.ts [hunk 2]
```diff
@@ -307,25 +388,28 @@ export class AngleSweep implements BeJSONFunctions {
...
   }
-  /** test if start and end angles match, with explicit name to clarify that there is no test for 360-degree shifts. */
+  /** test if start and end angles match with radians tolerance.
+   * * Period shifts are allowed.
+   */
   public isAlmostEqualAllowPeriodShift(other: AngleSweep): boolean {
     return Angle.isAlmostEqualRadiansAllowPeriodShift(this._radians0, other._radians0)
-      && Angle.isAlmostEqualRadiansNoPeriodShift(this._radians1 - this._radians0, other._radians1 - other._radians0);
+      && Angle.isAlmostEqualRadiansAllowPeriodShift(this._radians1 - this._radians0, other._radians1 - other._radians0);
   }
```
Comment (dassaf4) [inline core/geometry/src/geometry3d/AngleSweep.ts:399]: We need to confirm with Earlin, but I believe `AngleSweep` "equality" means the start angle is optionally allowed to shift by 2pi, but _never_ the sweep. So I'm thinking the implementation should remain untouched, but please clarify this subtlety in the documentation.

**Review by dassaf4 — APPROVED** [review]

**Comment by dassaf4** [comment]
Looks good, modulo my 2 comments.  I'll go ahead and approve.

File: core/geometry/src/geometry3d/AngleSweep.ts (hunk 2 above)
Comment (saeeedtorabi) [reply core/geometry/src/geometry3d/AngleSweep.ts:399]: will ask him

**Review by mdastous-bentley — COMMENTED** [review]
Why so many changelogs are getting removed?

**Comment by pmconne** [comment]
> Why so many changelogs are getting removed?

GitHub bugged out. Fixed.

**Review by dassaf4 — APPROVED** [review]

**Pull request merged**